src/controllers/player.controller.js

```diff
@@ -0,0 +1,48 @@
+import GameBoardController from './game-board.controller';
+import AppService from '../app.service';
+
+const PlayerController = (n) => {
+  const board = GameBoardController(n === 'AI' ? 'AI' : 'Human');
+  const appService = AppService();
+  const name = n;
+
+  const aiPlaceShips = () => {
+    // TODO: Remove
+    board.placeShip(0, 0, 3, false);
+
+    // let shipBlocks = 20;
```

Comment:
**Kindly remove commented code inside file, you can always use git history to access previous code right**:slightly_smiling_face: